src/relay/backend/compile_engine.cc

```diff
@@ -770,7 +770,7 @@ class CompileEngineImpl : public CompileEngineNode {
 };
 
 /*! \brief The global compile engine */
-const CompileEngine& CompileEngine::Global() {
+CompileEngine& CompileEngine::Global() {
```

Comment:
What's the reason for dropping the `const` here?

Reply:
Clearing the cache mutates the `CompileEngine`, so in order to clear the cache after each build we need to modify the global state.